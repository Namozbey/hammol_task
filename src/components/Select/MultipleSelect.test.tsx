import React from "react";
import { render } from "@testing-library/react";
import { MultipleSelect } from ".";

const options = [
  { label: "All", value: "all" },
  { label: "Docs", value: "doc" },
];

describe("Testing Multiple Select", () => {
  it("Snapshots", () => {
    const { asFragment } = render(
      <MultipleSelect value={["doc"]} options={options} />
    );
    expect(asFragment()).toMatchSnapshot();
  });
});
