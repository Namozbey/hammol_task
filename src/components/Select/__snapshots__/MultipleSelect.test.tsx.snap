// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing Multiple Select Snapshots 1`] = `
<DocumentFragment>
  <div
    class="sc-bczRLJ eRPedS"
  >
    <div
      class="sc-gsnTZi eIUvRC"
    >
      Select Genre
    </div>
    <span
      class="sc-dkzDqf ewRrEr material-icons"
    >
      arrow_drop_down
    </span>
    <div
      class="sc-hKMtZM bATtUK"
    >
      <label
        class="sc-eCYdqJ gYrPwk"
        font-size="20px"
      >
        <input
          class="sc-ftvSup dGIJev"
          name="all"
          type="checkbox"
        />
        <p>
          All
        </p>
      </label>
      <label
        class="sc-eCYdqJ gYrPwk"
        font-size="20px"
      >
        <input
          checked=""
          class="sc-ftvSup dGIJev"
          name="doc"
          type="checkbox"
        />
        <p>
          Docs
        </p>
      </label>
    </div>
  </div>
</DocumentFragment>
`;
